controller/env/broker.go

```diff
@@ -209,6 +286,10 @@ func (b *Broker) sendApiSessionCreates(apiSession *persistence.ApiSession) {
 		return
 	}
 
+	if len(fingerprints) == 0 {
+		return
```

Comment:
not even a debug here? feels like the sort of thing that might 'maybe' be useful some day? if not - i trust you :)

Reply:
Happens quite often for username/password logins. They don't have any certs. This line is specifically meant to weed them out.